### PR TITLE
docs(card): improve tabProps link

### PR DESCRIPTION
**Pull request by hengkx**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

-----
[View rendered components/card/index.en-US.md](https://github.com/ant-design/ant-design/blob/docs-card/components/card/index.en-US.md)
[View rendered components/card/index.zh-CN.md](https://github.com/ant-design/ant-design/blob/docs-card/components/card/index.zh-CN.md)

### CODE REVIEW

File: components/card/index.en-US.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ A card can be used to display content related to a single subject. The content c
 | title | Card title | ReactNode | - |  |
 | type | Card style type, can be set to `inner` or not set | string | - |  |
 | onTabChange | Callback when tab is switched | (key) => void | - |  |
-| tabProps | [Tabs](/components/tabs/#Tabs) | - | - |  |
+| tabProps | [Tabs](/components/tabs/#API) | - | - |  |
```
Comment (afc163) [inline components/card/index.en-US.md:41]: 原来的好像没什么问题。

Comment (hengkx) [reply components/card/index.en-US.md:41]: 有问题 你试试 长屏幕 看下，而且 tab里面 也没有 `Tabs`这个锚点吧

Comment (hengkx) [reply components/card/index.en-US.md:41]: 看错了 有 锚点

Comment (hengkx) [reply components/card/index.en-US.md:41]: 还是定位的  问题

Comment (afc163) [reply components/card/index.en-US.md:41]: 有。

image

Comment (hengkx) [reply components/card/index.en-US.md:41]: 不管是什么 都跳不过去🤣

Comment (afc163) [reply components/card/index.en-US.md:41]: 这确实是个问题。

**Pull request closed without merging**